packages/lwc/scripts/utils/rollup.js

```diff
@@ -5,46 +5,38 @@
  * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
  */
 const path = require('path');
+
 const { rollup } = require('rollup');
-const typescript = require('typescript');
 const rollupTypescriptPlugin = require('rollup-plugin-typescript');
-const rollupReplace = require('@rollup/plugin-replace');
 const { terser: rollupTerser } = require('rollup-plugin-terser');
-const babel = require('@babel/core');
-const babelFeaturesPlugin = require('@lwc/features/src/babel-plugin');
-const { generateTargetName } = require('./helpers');
+const rollupReplace = require('@rollup/plugin-replace');
+const { default: nodeResolvePlugin } = require('@rollup/plugin-node-resolve');
 
-function rollupFeaturesPlugin(prod) {
```

Comment:
There was no need to do the lwc feature transformation here. Each individual package is running this transform to generate the content in the `dist` directory. When `lwc` package build the input files are free of `@lwc/features` import.

Reply:
I don't think this is true. The way this system works is that all flags--regardless of whether they are runtime or compile-time--are transformed into runtime flags for non-prod to facilitate testing. For prod, we enable the tree-shaking of branches and code is actually stripped if the flag is set to false. Since the concept of prod/non-prod only exists in this distribution package, I'm guessing we need to keep this here.